mw-webapp/src/component/text/Text.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * Enum of text type styles
+ */
+export enum TextType {
+  DEFAULT = "default"
+}
+
+/**
+ * Text props
+ */
+interface TextProps {
+
+  /**
+   * Displayed text
+   */
+  text: string | number;
+
+  /**
+   * Displayed type theme text
+   * @default {@link TextType}
+   */
+  textType?: TextType.DEFAULT;
+
+  /**
+   * Data attributes for cypress testing
+   */
+  cy?: string;
+
+  /**
+   * Custom class for the Text.
+   */
+  className?: string;
+}
+
+/**
+ * Text component
+ */
+export const Text = (props: TextProps) => {
+  return (
+    <div
```

Comment:
Can you please to check is it possible to use tag p or span here instead of div?

Reply:
Replaced <div> with <p>. I also corrected the tests to render the <p> tag